### ISSUE TITLE
it also hang on during "Identifying TIGRFAM protein families." gtdbtk-2.3.2

### ISSUE THREAD

**Issue by Caiyulu-818** [Ecogenomics/GTDBTk#608]
Hi，
When i use the gtdbtk-2.1.0/bin/python3.8 or gtdbtk-2.3.2/bin/python3.8 gtdbtk de_novo_wf --genome_dir /public/home/lcy/arc/soil4210 --archaea --outgroup_taxon p__Altiarchaeota --out_dir /public/home/lcy/application/iphop_db/arc_MAGs_GTDB-tk_results1/ --cpus 5 --force --extension fa
because I need the GTDB database version release 214.
it also hangs on during "Identifying TIGRFAM protein families."

Looking forward for your reply ~

**Comment by pchaumeil** [comment]
Hello, 

How are you running Tk ? with Docker? on an HPC using Slurm or PBS? 
How many genomes are you running at once? Does the pipeline works with one genome only?

Thanks,
Pierre

**Comment by Caiyulu-818** [comment]
Hi,
I am running Tk with HPC using PBS. I input about 4000 genomes at once with CPU 64, and when I separated them into two pieces(2000 genomes) with 20 CPUs, it also failed, but when I input 2000 genomes with CPU 5, it succeeded.

I am just curious that I succeeded in inputting about 6W genomes using“gtdbtk classify”, but this time it failed just inputting 4000 genomes using "gtdbtk de_novo_wf ".

Best regards
Caiyu
